### PR TITLE
Feat: refactor codes

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
added :
1)Connected all the links to documentation page
2)fixed slider
3)fixed text utility

### CODE REVIEW

File: Utilities/Colors/color.html [hunk 1]
```diff
@@ -12,51 +12,54 @@
...
+          <h3>Get Started</h3>
+          <div class="divider"></div>
+          <ul>
+            <li><a href="/Documentation/documentation.html">Introduction</a></li>
+            <li><a class="sidebar--active" href="/Utilities/Colors/color.html">Colors</a></li>
+            <li><a href="/Utilities/Text Utility/textUtil.html">Text Utility </a></li>
+            <li><a href="/Utilities/Conclusion/conclusion.html">Conclusion</a></li>
+            
+          
+
```
Comment (JantuDeb) [inline Utilities/Colors/color.html:36]: remove unnecessary space.

File: Utilities/Conclusion/conclusion.html [hunk 2]
```diff
@@ -13,52 +13,54 @@
...
+      <h3>Get Started</h3>
+      <div class="divider"></div>
+      <ul>
+        <li><a href="/Documentation/documentation.html">Introduction</a></li>
+        <li><a href="/Utilities/Colors/color.html">Colors</a></li>
+        <li><a href="/Utilities/Text Utility/textUtil.html">Text Utility </a></li>
+        <li><a class="sidebar--active" href="/Utilities/Conclusion/conclusion.html">Conclusion</a></li>
+        
+      
+
```
Comment (JantuDeb) [inline Utilities/Conclusion/conclusion.html:37]: same here

**Review by JantuDeb — APPROVED** [review]
Looks good to me

File: Utilities/Conclusion/conclusion.html (hunk 2 above)
Comment (SuyashTiwariCoder) [reply Utilities/Conclusion/conclusion.html:37]: ok,

File: Utilities/Colors/color.html (hunk 1 above)
Comment (SuyashTiwariCoder) [reply Utilities/Colors/color.html:36]: ok i will remove it

**Review by dev-enforced — APPROVED** [review]
Looks good to merge!!Good use of refactor

**Review by dikshadhote — APPROVED** [review]
You can merge it !

**Pull request merged**